site/docs/components/slider/examples.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+<LivePreviewControls>
+
+<LivePreview componentName="slider" exampleName="Single">
+
+## Single Slider
+
+Selection of a single value from a horizontally oriented slider track. Values can be displayed either inline or below the track.
+
+A tooltip is displayed when the thumb is dragged to ensure that incremental values remain visible when the number increments are not displayed along the track (i.e., with vertical label position).
+
+</LivePreview>
+
+<LivePreview componentName="slider" exampleName="Range">
+
+## Range Slider
+
+Selection of a range of values from a horizontally oriented slider track. This determines a minimum and maximum range. Values can be displayed either inline or below the track.
+
+### Best practices
+
+The default minimum range is set to 1 step, therefore thumbs cannot pass or overlap.
+
+</LivePreview>
+
+<LivePreview componentName="slider" exampleName="CustomStep">
+
+## Custom Step
+
+By default, the slider step increment is set to 1. This can be customised by configuring the ‘step’ prop. We recommend using an increment most suitable to your use case by using a step relative to the granularity of the scale. This prevents slider selections from becoming tedious and difficult to manage.
```

Comment:
```suggestion
By default, the slider step increment is set to 1. This can be customized using the ‘step’ prop. We recommend using an increment most suitable to your use case by using a step relative to the granularity of the scale. This prevents slider selections from becoming tedious and difficult to manage.
```